doc/agentservice.md

```diff
@@ -0,0 +1,11 @@
+# Agent service
+Agent service is responsible for serving collected data to the user. Once user create a source for his vCenter environment the Agent service provide a streaming service to download OVA image that is ready to be booted on the vCenter enviroment to run the collection of the data.
+
+## Agent API
+There are two APIs related to the Agent.
+
+### Internal API
+Internal Agent API exposed for the UI. This API contains operations to create source, download OVA, etc. By default running on port 3443. This API is not exposed externaly to users, it's used only internally by UI.
+
+### Agent API
+The Agent API is exposed for the communication with the agent. The only operation is to update the status of the source. By default running on port 7443. This API must be externally exposed, so agent VM can send over data.
```

Comment:
I think that the terms used here are a bit confusing.
In the headers you used the following terms:
* Agent service
* Agent Virtual Machine (or AgentVM)

But in this sentence you use the term "agent" (in: `for the communication with the agent`), which I assumes it reffers to the "AgentVM"
We need make sure we use the same terms in all places in order to keep it as clear as possible